File: httpsnippet/src/test/resources/output/java/okhttp/full.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
  .url("http://mockbin.com/har?key=value")
  .post(null)
  .addHeader("Cookie", "foo=bar;bar=baz")
  .addHeader("content-type", "application/x-www-form-urlencoded")
  .addHeader("accept", "application/json")
  .build();

Response response = client.newCall(request).execute();

File: src/test/resources/output/csharp/restsharp/full.cs
var client = new RestClient("http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value");
var request = new RestRequest(Method.POST);
request.AddHeader("accept", "application/json");
request.AddHeader("content-type", "application/x-www-form-urlencoded");
request.AddCookie("foo", "bar");
request.AddCookie("bar", "baz");
request.AddParameter("application/x-www-form-urlencoded", "foo=bar", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
